shared/haml/course_explorer_table.haml

```diff
@@ -20,11 +20,141 @@
+        <li> <strong>Languages:</strong> English only </li>
+      </ul>'
+  }
+
+  courses << {
+    name: "CS Discoveries",
+    starts: 7,
+    ends: 9,
+    subtitle: "Semester or full year course",
+    link: CDO.code_org_url("/educate/csd"),
+    course_link: CDO.studio_url("/courses/csd"),
+    img: "/shared/images/fit-500/courses/logo_csd.jpg",
+    description: 'CS Discoveries is an introductory course that empowers students to engage with Computer Science as a medium for creativity, communication, problem solving, and fun.
+      <ul>
+        <li> <strong>Length:</strong> Semester or full-year course (? hours) </li>
+        <li> <strong>Recommended age:</strong> Ages 11-14 (6th through 9th grade) </li>
+        <li> <strong>Prerequisites:</strong> None </li>
+        <li> <strong>Optional professional learning:</strong> Free 1-day workshop, in person throughout the US and   line globally. </li>
+        <li> <strong>Cost to use Curriculum:</strong> None </li>
+        <li> <strong>Languages:</strong> English only </li>
+      </ul>'
+  }
+
+  courses << {
+    name: "CS Fundamentals",
+    starts: 0,
+    ends: 5,
+    subtitle: "20 hour courses for each grade can be taught once a week",
+    link: CDO.code_org_url("/educate/curriculum/elementary-school"),
+    img: "/shared/images/courses/logo_tall_course1.jpg",
+    description: 'Designed to be fun and engaging, Code.org’s progression of CS Fundamentals courses blend online and "unplugged" non-computer activities to teach students computational thinking, problem solving, programming concepts and digital citizenship.
+      <ul>
+        <li> <strong>Length:</strong> 15-20 hours per course, 5 courses </li>
+        <li> <strong>Recommended age:</strong> Ages 6-11 (Kindergarten through 5th grade) </li>
+        <li> <strong>Prerequisites:</strong> None </li>
+        <li> <strong>Optional professional learning:</strong> Free 1-day workshop, in person in the US and   line globally. </li>
+        <li> <strong>Cost to use curriculum:</strong> None </li>
+        <li> <strong>Languages:</strong> English, Spanish, +24 more </li>
+      </ul>'
+  }
+
+  courses << {
+    name: "Pre-reader Express",
+    starts: 0,
+    ends: 2,
+    subtitle: "",
+    link: CDO.studio_url("/s/pre-express"),
+    img: "/shared/images/courses/logo_tall_pre-express.jpg",
+    description: 'This single condensed 14-hour course covers the core concepts from the kindergarten and first grade courses in CS Fundamentals at an accelerated pace.
+      <ul>
+        <li> <strong>Length:</strong> 14 hours </li>
+        <li> <strong>Recommended age:</strong> Ages 4-8 </li>
+        <li> <strong>Prerequisites:</strong> None </li>
+        <li> <strong>Optional professional learning:</strong> Free 1-day workshop, in person in the US and online globally. </li>
+        <li> <strong>Cost to use curriculum:</strong> None </li>
+        <li> <strong>Languages:</strong> English</li>
```

Comment:
English only 

(My bad. Should be consistent with CSD/CSP so better to add the word only.)